engines/gob/gob.cpp

```diff
@@ -699,9 +699,10 @@ void GobEngine::deinitGameParts() {
 }
 
 Common::Error GobEngine::initGraphics() {
-	if        (is800x600()) {
-		warning("GobEngine::initGraphics(): 800x600 games currently unsupported");
-		return Common::kUnsupportedGameidError;
+	if (is800x600()) {
```

Comment:
All the games that you marked as 800x600 are currently unsupported, so why this change?

Reply:
or should i make a new PR to use the correct resolution only in the tables with the 800x600 implementation?

Reply:
The purpose of this check is to stop execution for 800x600 games (because they are unsupported) and print a warning. With your change you allow the screen initialization but does any of these games actually work?

Reply:
i thought because these games are using already 800x600, so i can allow already 800x600 in the Engine for later purposes.

Reply:
nope Adibou 1 is 320x200 and Adibou 640x480